Pod/Classes/public/error/Error.swift

```diff
@@ -0,0 +1,150 @@
+//
+//  Error.swift
+//  Pods
+//
+//  Created by Jake Young on 6/14/16.
+//
+
+import Foundation
+
+/// Debugging levels for WistiaKit
+public enum DebuggingLevel {
+    
+    /// Shhh, fust be silent.
+    case None
+    
+    /// Print the high level activity and errors.
+    case Friendly
+    
+    /// Print all the things.
+    case Annoying
+}
+
+internal func printErrorForDebuggingLevel(debuggingLevel debuggingLevel: DebuggingLevel, message: String? = nil, error: NSError? = nil) {
```

Comment:
@spinosa after revisiting this the debugging levels are nice to have, but the real need is to print our what's going on or not, so resubmitting with debug boolean, and applying Swift 3 guidelines.